frontend/demo/component/badge/badge-interactive.ts

```diff
@@ -1,22 +1,108 @@
-import 'Frontend/demo/init'; // hidden-full-source-line
+import { getPeople } from 'Frontend/demo/domain/DataService'; // hidden-full-source-line
+import 'Frontend/demo/init';
 
-import { html, LitElement, customElement } from 'lit-element';
-import '@vaadin/vaadin-checkbox/vaadin-checkbox';
+import '@vaadin/vaadin-button';
+import '@vaadin/vaadin-combo-box';
+import '@vaadin/vaadin-icons/vaadin-icons';
+import '@vaadin/vaadin-lumo-styles/icons';
```

Comment:
Unnecessary import of Vaadin icons since only Lumo icons is used here.
```suggestion
import '@vaadin/vaadin-lumo-styles/icons';
```